languages/translations/en.json

```diff
@@ -125,7 +125,8 @@
         "depart": {
             "default": {
                 "default": "Head {direction}",
-                "name": "Head {direction} on {way_name}"
+                "name": "Head {direction} on {way_name}",
+                "namedistance": "Head {direction} on {way_name} for {distance}"
```

Comment:
In the absence of #155, do we need to also add this line to all the other translation files?

Reply:
hmmm... if we're going to do that, I'd like to use `depart name` rather than the english here.

Reply:
I did this!